src/Container/AgFeatureGrid/AgFeatureGrid.jsx

```diff
@@ -0,0 +1,1021 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { AgGridReact } from 'ag-grid-react';
+import {
+  isEqual,
+  isFunction,
+  kebabCase
+} from 'lodash';
+import OlStyle from 'ol/style/style';
+import OlStyleFill from 'ol/style/fill';
+import OlStyleCircle from 'ol/style/circle';
+import OlStyleStroke from 'ol/style/stroke';
+import OlMap from 'ol/map';
+import OlFeature from 'ol/feature';
+import OlSourceVector from 'ol/source/vector';
+import OlLayerVector from 'ol/layer/vector';
+import OlGeomGeometry from 'ol/geom/geometry';
+import OlGeomGeometryCollection from 'ol/geom/geometrycollection';
+
+import { MapUtil } from '../../index';
+import { CSS_PREFIX } from '../../constants';
+
+import './AgFeatureGrid.less';
+import 'ag-grid/dist/styles/ag-grid.css';
+import 'ag-grid/dist/styles/ag-theme-balham.css';
+
+/**
+ * The AgFeatureGrid.
+ *
+ * @class The AgFeatureGrid
+ * @extends React.Component
+ */
+export class AgFeatureGrid extends React.Component {
+
+  /**
+   * The className added to this component.
+   * @type {String}
+   * @private
+   */
+  _className = `${CSS_PREFIX}ag-feature-grid`
+
+  /**
+   * The class name to add to each table row.
+   * @type {String}
+   * @private
+   */
+  _rowClassName = `${CSS_PREFIX}ag-feature-grid-row`;
+
+  /**
+   * The prefix to use for each table row class.
+   * @type {String}
+   * @private
+   */
+  _rowKeyClassNamePrefix = 'row-key-';
+
+  /**
+   * The hover class name.
+   * @type {String}
+   * @private
+   */
+  _rowHoverClassName = 'ag-row-hover';
+
+  /**
+   * The source holding the features of the grid.
+   * @type {ol.source.Vector}
+   * @private
+   */
+  _source = null;
+
+  /**
+   * The layer representing the features of the grid.
+   * @type {ol.layer.Vector}
+   * @private
+   */
+  _layer = null;
+
+  /**
+   * The properties.
+   * @type {Object}
+   */
+  static propTypes = {
+    /**
+     * An optional CSS class which should be added to the table.
+     * @type {String}
+     */
+    className: PropTypes.string,
+
+    /**
+     * The height of the grid.
+     *
+     * @type {Number|String}
+     */
+    height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
```

Comment:
Why do we need this?

Reply:
This fits the default ag-grid behaviour to match its parent div's height and width. I'm sure there are other/better ways to do this, if you have a specific approach in mind.

Reply:
Not yet. Just go ahead with the `width` and `height`...